src/ServiceControl.Audit.AcceptanceTests/Auditing/When_processed_message_is_imported.cs

```diff
@@ -70,19 +70,18 @@ public async Task Should_be_accessible_via_the_rest_api()
                 Assert.That(auditedMessage.TimeSent, Is.Not.EqualTo(DateTime.MinValue), "Time sent should be correctly set");
                 Assert.That(auditedMessage.ProcessedAt, Is.Not.EqualTo(DateTime.MinValue), "Processed At should be correctly set");
 
-                Assert.That(TimeSpan.Zero, Is.LessThan(auditedMessage.ProcessingTime), "Processing time should be calculated");
+                Assert.That(auditedMessage.ProcessingTime, Is.LessThan(TimeSpan.Zero), "Processing time should be calculated");
+                Assert.That(auditedMessage.CriticalTime, Is.LessThan(TimeSpan.Zero), "Critical time should be calculated");
+
+                Assert.That(auditedMessage.MessageIntent, Is.EqualTo(MessageIntent.Send), "Message intent should be set");
             });
-            Assert.That(TimeSpan.Zero, Is.LessThan(auditedMessage.CriticalTime), "Critical time should be calculated");
-            Assert.That(auditedMessage.MessageIntent, Is.EqualTo(MessageIntent.Send), "Message intent should be set");
```

Comment:
I think these assertions were wrong in the first place and should be fixed. Expected and actual was mixed up and I think the condition should be that both times are GreaterThanZero

Reply:
Addressed now, sorry I missed this one